cmd/flowrunner/runner_test.go

```diff
@@ -168,27 +170,53 @@ func newTestHTTPServer() *httptest.Server {
 	return server
 }
 
+type fixedUUIDGenerator struct {
+	uuids []string
+	index int
+}
+
+func fixedUUIDGeneratorFromFile(path string) (*fixedUUIDGenerator, error) {
+	content, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	return &fixedUUIDGenerator{
+		uuids: strings.Split(string(content), "\n"),
+		index: 0,
+	}, nil
+}
+
+func (g *fixedUUIDGenerator) Next() string {
+	if g.index >= len(g.uuids) {
+		panic("Fixed UUID list for testing exhausted. Add more!")
+	}
+
+	u := g.uuids[g.index]
+	g.index++
+	return u
+}
+
 func TestFlows(t *testing.T) {
 	server := newTestHTTPServer()
 	server.Start()
 	defer server.Close()
 
+	uuids, err := fixedUUIDGeneratorFromFile("testdata/uuids.list")
+	require.NoError(t, err)
+
+	utils.SetUUIDGenerator(uuids)
+	defer utils.SetUUIDGenerator(utils.RandomUUID4Generator{})
+
 	// save away our server URL so we can rewrite our URLs
 	serverURL = server.URL
 
 	for _, test := range flowTests {
 		testJSON, err := readFile("flows/", test.output)
-		if err != nil {
-			t.Errorf("Error reading output file for flow '%s' and output '%s': %s", test.assets, test.output, err)
-			continue
-		}
+		require.NoError(t, err, "Error reading output file for flow '%s' and output '%s': %s", test.assets, test.output, err)
```

Comment:
`require.NoError` is a clean way to fail fast on things that aren't really test failures so much as fundamental problems